plugins/evernote.koplugin/main.lua

```diff
@@ -59,7 +70,10 @@ function EvernoteExporter:isDocless()
 end
 
 function EvernoteExporter:readyToExport()
-    return self.evernote_token ~= nil or self.html_export ~= false or self.txt_export ~= false
+    return self.evernote_token ~= nil or 
```

Comment:
There's a lot of trailing whitespace in the diff.
```suggestion
    return self.evernote_token ~= nil or
```
(Personally I prefer to start the line with or but whichever you prefer.)